### PR TITLE
[16.0][MIG] partner_contact_role: Migration to 16.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
@qrtl QT4695

### CODE REVIEW

File: partner_contact_role/views/res_partner_role.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <tree editable="bottom">
+                <field name="name" />
+                <field name="active" widget="boolean_toggle" />
+           </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="res_partner_role_act_window">
+        <field name="name">Partner Role</field>
+        <field name="res_model">res.partner.role</field>
+        <field name="view_mode">tree</field>
```
Comment (kanda999) [inline partner_contact_role/views/res_partner_role.xml]:
```suggestion
        <field name="view_mode">tree,form</field>
```
Someone might be depending on this module to extend the form view, so we will keep the form view

File: partner_contact_role/views/res_partner_role.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Camptocamp
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="res_partner_role_form_view">
+        <field name="name">res.partner.role.tree</field>
```
Comment (kanda999) [inline partner_contact_role/views/res_partner_role.xml]:
```suggestion
        <field name="name">res.partner.role.form</field>
```

**Review by kanda999 — COMMENTED** [review]
Could you please update the comment of this PR?

**Review by kanda999 — APPROVED** [review]
Function test.

**Review by yostashiro — COMMENTED** [review]
@AungKoKoLin1997 As we just talked, let's separate the commits for migration and the enhancement (adding the active field).

**Review by yostashiro — APPROVED** [review]
Code review.

**Comment by AungKoKoLin1997** [comment]
@oca/partner-contact-maintainers 
Can you please take a look this migration PR?

**Review by yajo — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**